be/src/storage/olap_server.cpp

```diff
@@ -122,6 +122,11 @@ Status StorageEngine::start_bg_threads() {
             compaction_scheduler.schedule();
         });
         Thread::set_thread_name(_compaction_scheduler, "compact_sched");
+        LOG(INFO) << "compaction scheduler started";
```

Comment:
```
LOG(INFO) << "compaction scheduler started";
```
The log is non-sense.

Reply:
it can help to locate the background task start progress. I tend to keep it.